client/src/utils/trackEvents.js

```diff
@@ -0,0 +1,20 @@
+const ReactGA = require('react-ga');
+
+const trackingId = 'G-KVV9QJKG5K';
```

Comment:
you could replace this line with
`const trackingId = process.env.GOOGLE_ANALYTICS_TRACKING_ID;`

Reply:
Then add in .env.example file. Usually we put sensitive information (such as credential, etc) into environment variable. 
GOOGLE_ANALYTICS_TRACKING_ID=G-KVV9QJKG5K

Reply:
Thank you very much for the suggestions. The code has been updated accordingly.